runtime/memprof.c

```diff
@@ -629,6 +629,9 @@ static memprof_orphan_table_t orphans = NULL;
 /* lock controlling access to `orphans` variable */
```

Comment:
... and writes to `orphans_flag` (I think).

Reply:
Done